src/plugins/index_pattern_management/public/management_app/mount_management_section.tsx

```diff
@@ -0,0 +1,193 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { HashRouter, Switch, Route } from 'react-router-dom';
+
+import { i18n } from '@kbn/i18n';
+import { I18nProvider } from '@kbn/i18n/react';
+import { StartServicesAccessor } from 'src/core/public';
+import { IIndexPattern } from 'src/plugins/data/public';
+
+import { ManagementAppMountParams } from '../../../management/public';
+import {
+  IndexPatternTableWithRouter,
+  EditIndexPatternContainer,
+  CreateEditFieldContainer,
+  CreateIndexPatternWizardWithRouter,
+} from '../components';
+import { IndexPatternManagementStartDependencies, IndexPatternManagementStart } from '../plugin';
+
+const appTitle = i18n.translate('indexPatternManagement.indexPattern.sectionsHeader', {
+  defaultMessage: 'Index Patterns',
+});
+const crumb = [{ text: appTitle }];
+
+const readOnlyBadge = {
+  text: i18n.translate('indexPatternManagement.indexPatterns.badge.readOnly.text', {
+    defaultMessage: 'Read only',
+  }),
+  tooltip: i18n.translate('indexPatternManagement.indexPatterns.badge.readOnly.tooltip', {
+    defaultMessage: 'Unable to save index patterns',
+  }),
+  iconType: 'glasses',
+};
+
+export async function mountManagementSection(
+  getStartServices: StartServicesAccessor<IndexPatternManagementStartDependencies>,
+  params: ManagementAppMountParams
+) {
+  params.setBreadcrumbs(crumb);
+  const [
+    { chrome, application, savedObjects, uiSettings, notifications, overlays, http, docLinks },
+    { data },
+    indexPatternManagementStart,
+  ] = await getStartServices();
+  const defaultIndex = uiSettings.get('defaultIndex');
+  const canSave = application.capabilities.indexPatterns.save as boolean;
+
+  if (!canSave) {
+    chrome.setBadge(readOnlyBadge);
+  }
+
+  const indexPatterns =
```

Comment:
it's a reason why we still have one broken CI test: Please see that we build the indexPatterns list only once on mount that section and than pass it into all of our child components. 

As a result, if the user creates a new index template and goes back to the list page, the index will not be in this list =( 

I think instead of passing indexPattern (it also confuse a little cause we already have a indexPatterns service) we should pass savedObjects as dependencies and move that logic into util method to execute it from each route